api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -155,7 +155,7 @@ union all
 select 0 as id,3000 as analysis_id,CAST(vo1.visit_source_concept_id AS STRING) as stratum_1,'Visit' as stratum_3,
 COUNT(distinct vo1.PERSON_ID) as count_value,COUNT(distinct vo1.PERSON_ID) as source_count_value
 from \`${BQ_PROJECT}.${BQ_DATASET}.visit_occurrence\` vo1
-where vo1.visit_concept_id != vo1.visit_source_concept_id
+where vo1.visit_source_concept_id not in (select distinct visit_concept_id from \`${BQ_PROJECT}.${BQ_DATASET}.visit_occurrence\`)
```

Comment:
You hadn't modified these queries on the previous pass - double checking this change was intentional

Reply:
Yeah. Just thought the new check would do better job than the previous one
(in case the source concept is not equal to concept in this row but it is standard concept in some other row)